kbase-extension/scss/partials/_bulkImportCell.scss

```diff
@@ -60,74 +62,76 @@ $panel-separator-color: rgb(33, 150, 243);
 
   // kb-bulk-import__tab_pane_widget
   &__tab_pane_widget {
-    flex: 1;
-    padding: 4px;
-    margin: 15px;
+    padding: $tab-widget-padding;
+    margin: $tab-widget-margin;
   }
 
-  // kb-bulk-import__filetype_panel
-  &__filetype_panel {
-    background-color: use-color('base-lightest');
-    width: 24rem;
-
-    &__header {
-      padding: 8px;
-      border-bottom: 1px solid $component-border;
-      @include font-h4;
-
-      &_icon {
-        padding: 8px 6px 8px 8px;
-        color: rgb(0 0 0);
-      }
+  // kb-bulk-import-configure
+  &-configure {
+    &__container {
+      display: flex;
+      margin: -($tab-widget-margin + $tab-widget-padding);
```

Comment:
Since the filetype panel is now part of a tab pane widget, and all of those have some margin and padding, this pulls it back over to the upper-left corner to keep the chooser column flush. The actual input/params area gets set with those margins later.